github-actions/oss-readme-template/__tests__/workflows/valid-sdk-template.md

```diff
@@ -6,12 +6,12 @@
 
 My Awesome Requirements
 
-### Installing Momento and Running the Example
+### Installation
 
 My Awesome Examples
 
-### Using Momento
+### Usage
 
-My Awesome Hello World Code
+{{ ossUsageExamplePath }}
```

Comment:
See my other comments about slurping in the code to insert into the test template, but I think you want this to look something like:

```
### Usage

Checkout our [examples](./examples/README.md) directory for complete examples of how to use the SDK.

Here is a quickstart you can use in your own project:
\`\`\`
{{ usageExampleCode }}
\`\`\`

```

Reply:
ignore those backslashes in front of the backticks, I didn't know how to get github to render the triple-backticks inside of a code block.